container/docker/handler.go

```diff
@@ -83,18 +83,18 @@ type dockerContainerHandler struct {
 	fsHandler fsHandler
 }
 
-func getRwLayerID(containerID, storageDriverDir string, dockerVersion []int) (string, error) {
+func getRwLayerID(containerID, storageDir string, sd storageDriver, dockerVersion []int) (string, error) {
 	const (
 		// Docker version >=1.10.0 have a randomized ID for the root fs of a container.
 		randomizedRWLayerMinorVersion = 10
 		// Directory where the file containinig the randomized ID of the root fs of a container is stored in versions >= 1.10.0
-		rwLayerIDDir  = "../image/aufs/layerdb/mounts/"
-		rwLayerIDFile = "mount-id"
+		rwLayerIDDirTemplate = "image/%s/layerdb/mounts/"
```

Comment:
nit / optional:
there seems to be a widely copied / pasted way of doing this in github.  May want to use the standard approach:

https://github.com/search?q=%2Flayerdb%2Fmounts%2F&type=Code&utf8=%E2%9C%93